### PR TITLE
chore: fix remaining typing

### PR DESCRIPTION
**Pull request by chemelli74**

### CODE REVIEW

File: midealocal/security.py [hunk 1]
```diff
@@ -80,7 +80,7 @@ def set_aes_keys(self, key: bytes | str, iv: bytes | str) -> None:
         self._aes_key = key
         self._aes_iv = iv
 
-    def aes_encrypt_with_fixed_key(self, data: str) -> bytes:
+    def aes_encrypt_with_fixed_key(self, data: bytes) -> bytes:
```
Comment (rokam) [inline midealocal/security.py:83]: I think this change is creating the test error.

Comment (chemelli74) [reply midealocal/security.py:83]: I don't think so, I'm 99% sure its this one:

```
  self._security.aes_decrypt_with_fixed_key(response["key"]), b"0"
```

Comment (rokam) [reply midealocal/security.py:83]: response["key"] is str type.  Or you convert it prior calling or after.

**Review by rokam — APPROVED** [review]
that whould do it

**Pull request merged**